deployment/sklearn/sklearn-housing-gb-regression.ipynb

```diff
@@ -0,0 +1,448 @@
+      },
+      "outputs": [],
+      "source": [
+        "predictions = model.predict(X_test)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "NdgxciUI-Ak_"
+      },
+      "outputs": [],
+      "source": [
+        "MAE = mean_absolute_error(y_test, predictions)\n",
+        "MSE = mean_squared_error(y_test, predictions)\n",
+        "RMSE = mean_squared_error(y_test, predictions, squared=False) "
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "UvntJmfckF9V",
+        "outputId": "565750b8-1ba3-4f99-aabe-524736d82a76"
+      },
+      "outputs": [],
+      "source": [
+        "print(f\"MAE: {round(MAE, 3)}\")\n",
+        "print(f\"MSE: {round(MSE, 3)}\")\n",
+        "print(f\"RMSE: {round(RMSE, 3)}\")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "PQFxkzLijliO"
+      },
+      "source": [
+        "## 4. Verta Set Up"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "EIPPcopriB0G"
+      },
+      "outputs": [],
+      "source": [
+        "os.environ['VERTA_HOST'] = ''\n",
+        "os.environ['VERTA_EMAIL'] = ''\n",
+        "os.environ['VERTA_DEV_KEY'] = ''"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "bHgtbAo_6O82",
+        "outputId": "d452c390-d68f-411a-f56f-89b63a8f5393"
+      },
+      "outputs": [],
+      "source": [
+        "client = Client(os.environ['VERTA_HOST'])"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "vO9dZdFCkyc6"
+      },
+      "source": [
+        "## 5. Model Register"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "NaUrxGIkkxMV"
+      },
+      "outputs": [],
+      "source": [
+        "with open('model.pkl', 'wb') as f:\n",
+        "    cloudpickle.dump(model, f)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "3aZfYGUnlJ1p"
+      },
+      "outputs": [],
+      "source": [
+        "class HousePricePredictor(VertaModelBase):\n",
+        "    def __init__(self, artifacts):\n",
+        "        self.model = cloudpickle.load(open(artifacts['serialized_model'], 'rb'))\n",
+        "        \n",
+        "    @verify_io\n",
+        "    def predict(self, data):\n",
+        "        results = []\n",
+        "\n",
+        "        for item in data:\n",
+        "            results.append(self.model.predict(item).tolist())\n",
+        "        \n",
+        "        return results"
+      ]
```

Comment:
Also add `def example()` to this model if you can!

Reply:
Done.